com.unity.multiplayer.mlapi/Runtime/Messaging/MessagePacker.cs

```diff
@@ -19,7 +19,7 @@ internal static class MessagePacker
         // This method is responsible for unwrapping a message, that is extracting the messagebody.
         // Could include decrypting and/or authentication.
         internal static BitStream UnwrapMessage(BitStream inputStream, ulong clientId, out byte messageType, out SecuritySendFlags security)
-        {      
+        {
```

Comment:
I think changes in this file are not necessary — let's keep this file untouched.

Reply:
Well, @mattwalsh-unity requested the comment on the magic number 6 in the previous PR, which is why I changed the file. I don't really have an opinion either way. Matt & Fatih, can you reach an agreement whether it's worth touching the file, just for a comment?

Reply:
oh okay, I can take my word back, it's fine.

Reply:
Love the comment!